cli/Makefile

```diff
@@ -8,40 +8,41 @@ GO_FLAGS += "-ldflags=-s -w"
 # Avoid embedding the build path in the executable for more reproducible builds
 GO_FLAGS += -trimpath
 
-SRC_FILES = $(shell find . -name "*.go" | grep -v "_test.go")
+GO_FILES = $(shell find . -name "*.go")
+SRC_FILES = $(shell echo $GO_FILES | grep -v "_test.go")
 
-turbo: $(SRC_FILES) go.mod
+turbo: $(SRC_FILES) go.mod go.sum
 	CGO_ENABLED=0 go build $(GO_FLAGS) ./cmd/turbo
 
-# These tests are for development
-test: test-go vet-go
-
 ewatch: scripts/...
 	nodemon --exec "make turbo && make e2e" -e .ts,.go
 
 check-go-version:
 	@go version | grep ' go1\.18\.0 ' || (echo 'Please install Go version 1.18.0' && false)
 
 # This "TURBO_RACE" variable exists at the request of a user on GitHub who
-# wants to run "make test" on an unsupported version of macOS (version 10.9).
+# wants to run "make test-go" on an unsupported version of macOS (version 10.9).
 # Go's race detector does not run correctly on that version. With this flag
-# you can run "TURBO_RACE= make test" to disable the race detector.
+# you can run "TURBO_RACE= make test-go" to disable the race detector.
 TURBO_RACE ?= -race
 
-test-go:
-	go test $(TURBO_RACE) ./internal/...
+clean-go:
+	go clean -testcache ./...
+
+test-go: $(GO_FILES) go.mod go.sum
+	go test $(TURBO_RACE) ./...
 
-vet-go:
-	go vet ./cmd/... ./internal/...
+vet-go: $(GO_FILES) go.mod go.sum
+	go vet ./...
```

Comment:
Should we get rid of this entirely? It's covered under `golangci`. In fact, it might be better to replace it with a command to do `golangci-lint run --new-from-rev=main`

Reply:
Good call; I've updated that.